entity/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] To implement the `has-many` side of the relationship(`association`) between book and rental, a book has many rentals, a method for adding rentals should be present in the Book class. Something like so: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Please kindly fix that.